src/test/java/org/kohsuke/github/AbuseLimitHandlerTest.java

```diff
@@ -412,7 +412,6 @@ public void onError(IOException e, HttpURLConnection uc) throws IOException {
                         assertThat(uc.getContentType(), equalTo("application/json; charset=utf-8"));
                         assertThat(uc.getContentLength(), equalTo(-1));
                         assertThat(uc.getHeaderFields(), instanceOf(Map.class));
-                        assertThat(uc.getHeaderFields().size(), Matchers.greaterThan(25));
```

Comment:
I wonder why I even had this in there...

Reply:
Yeah, I think there's an opportunity to make some of the tests a bit more concise and flexible without losing the important sense checks about the mocks' behaviour or the validation of the actual code ... but I didn't want to change too much existing code in an unrelated PR. But I perhaps should have been a bit more conservative about how much I copy-pasted.